ci-operator/step-registry/aws/deprovision/route53/private-hosted-zone/aws-deprovision-route53-private-hosted-zone-commands.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+export AWS_SHARED_CREDENTIALS_FILE="${CLUSTER_PROFILE_DIR}/.awscred"
+REGION="${LEASED_RESOURCE}"
+HostedZoneId="${SHARED_DIR}/hosted_zone_id"
+
+echo "Deleting AWS route53 hosted zone"
+if [ ! -f "${HostedZoneId}" ]; then
+  echo "File ${HostedZoneId} does not exist."
+  exit 1
+fi
+
+cmd="aws --region $REGION route53 delete-hosted-zone --id $(cat ${HostedZoneId}) | jq -r '.ChangeInfo.Id' | awk -F '/' '{printf \$3}'"
```

Comment:
it looks like you're already doing the `jq`/`awk` bit in the creation step before writing to `hosted_zone_id` (and it makes sense to me to limit the `SHARED_DIR` data to information that other steps actually need).  Do you really need to do this additional JSON processing in the deprovision step?